### PR TITLE
✨ JS-614: Review Response Render

### PR DESCRIPTION
**Pull request by ct-aa**
# Related Tickets
- [JS-614](https://codeandtheory.atlassian.net/browse/JS-614)

# Description
Added the **Review Response Render** challenge solutions.

# Validation
* Open the Terminal and go to the directory of the **Review Response Render* folder.
* Run "node app.js".
* Visit http://localhost:3000/ on the browser.
* The browser should display the sample posts.

Screen Shot 2023-01-03 at 7 56 12 PM

# Risks
N/A

### CODE REVIEW

**Review by lycoangelo — CHANGES_REQUESTED** [review]
Hi @ct-aa, please remove the `node_modules`. It should be system specific.

**Comment by ct-aa** [comment]
> Hi @ct-aa, please remove the `node_modules`. It should be system specific.

Thanks, @lycoangelo. This is updated now.

File: Review Response Render/templates/main.pug [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+		meta(charset="UTF-8")
+		meta(http-equiv="X-UA-Compatible", content="IE=edge")
+		meta(name="viewport", content="width=device-width, initial-scale=1.0")
+		title Document
+	body 
+		section
+			ul
+				each post in posts
+					li= post.body
+
```
Comment (lycoangelo) [inline Review Response Render/templates/main.pug]: @ct-aa, let's change the indention from tabs to spaces and set it to 2 spaces. Also, please add a new line at the EOF.

**Review by lycoangelo — CHANGES_REQUESTED** [review]

File: Review Response Render/templates/main.pug (hunk 1 above)
Comment (ct-aa) [reply Review Response Render/templates/main.pug]: Thanks, @lycoangelo. This is now updated.

**Review by lycoangelo — APPROVED** [review]

**Review by lycoangelo — APPROVED** [review]

**Pull request merged**